### ISSUE TITLE
Parse error

### ISSUE THREAD

**Issue by jacepogi** [ninject/Ninject.Extensions.Wcf#18]
Hi , i manage to build and run the files but the following error occurs.

Inheritance security rules violated while overriding member: 'Ninject.Web.Common.NinjectHttpApplication.get_Kernel()'. Security accessibility of the overriding method must match the security accessibility of the method being overriden.

Would you know the cause and resolution for this? thank you :)
